### PR TITLE
List people who've agreed to be on the OTIAB

### PR DESCRIPTION
**Pull request by bhs**
(there are a few other invites outstanding, but this is good for now)

### CODE REVIEW

**Comment by bhs** [comment]
(per discussion at the call last Friday)

**Comment by bhs** [comment]
oh: once we've established this list, we can add them here: https://github.com/orgs/opentracing/teams/otiab

**Comment by wu-sheng** [comment]
Glad to have this.

File: project_organization.md [hunk 1]
```diff
@@ -32,15 +32,23 @@ OTSC meetings agenda and minutes are recorded in [this Google document](https://
 
 OpenTracing has an **Industrial Advisory Board** (OTIAB) that comprises engineers with detailed knowledge of tracing technology at companies of different scales and software maturity. The OTSC members engage with the OTIAB on a periodic basis to gather feedback about successes, challenges, and open-source packages which would benefit from greater OpenTracing support.
 
+The current OpenTracing Industrial Advisory Board is as follows (in alphabetical order):
+
+- Brian Hanafee: Wells Fargo
```
Comment (pavolloffay) [inline project_organization.md]: Maybe add github usernames as https://github.com/orgs/opentracing/teams/otiab as cannot be made public.

Comment (wu-sheng) [inline project_organization.md]: Good idea. 👍

**Comment by bhs** [comment]
Usernames added... I will merge this in the next day or two if there are no objections.

**Pull request merged**